### PR TITLE
Glossary: shorter indent for dictionary rendering

### PR DESCRIPTION
**Pull request by miurahr**
- resolve RFE#1588 and RFE#1587

Signed-off-by: Hiroshi Miura <miurahr@linux.com>

### CODE REVIEW

File: src/org/omegat/gui/glossary/DictionaryGlossaryRenderer.java [hunk 1]
```diff
@@ -57,8 +57,11 @@ public void render(GlossaryEntry entry, IRenderTarget<?> trg) {
         for (int i = 0; i < targets.length; i++) {
             if (i == 0 || (!targets[i].equals(targets[i - 1]))) {
                 if (hasComments) {
-                    trg.append("\n\t");
-                    hasComments = false;
+                    if (trg instanceof HtmlTarget) {
```
Comment (amake) [inline src/org/omegat/gui/glossary/DictionaryGlossaryRenderer.java:60]: This smells like a problem. We shouldn't have to know about the type of `trg`. The render target itself should know what to do here. Like maybe `IRenderTarget` should have methods like `startIndent` and `endIndent` or something?

Comment (t-cordonnier) [reply src/org/omegat/gui/glossary/DictionaryGlossaryRenderer.java:60]: Yes I agree with that. We cannot be sure that there will not be another renderer in the future.
Two possibilities: either methods return a string and we write trg.append(trg.startIndent()) or we have a method trg.appendStartIndent(), as you prefer

Comment (miurahr) [reply src/org/omegat/gui/glossary/DictionaryGlossaryRenderer.java]: Add `trg.appendStartIndent()`

**Comment by amake** [comment]
Thanks. I've squashed this to `master`.

**Pull request closed without merging**